### PR TITLE
[kueuectl] Add short description for first level pass-through commands.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add short description for first level pass-through commands.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/kueuectl/app/passthrough/passthrough.go [hunk 1]
```diff
@@ -27,53 +27,76 @@ import (
 
 type passThroughType struct {
 	name    string
+	short   string
```
Comment (trasc) [inline cmd/kueuectl/app/passthrough/passthrough.go:30]: https://github.com/kubernetes-sigs/kueue/pull/2992#discussion_r1765076221

Comment (mbobrovskyi) [reply cmd/kueuectl/app/passthrough/passthrough.go:30]: >  and define a different one for command if necessary (but for mentioning that it's a pass to kubectl is sufficient)/

Sorry, I missed it

But why we can't use the same struct? This is the same command but with second level. No? Or I missing something?

Comment (mbobrovskyi) [reply cmd/kueuectl/app/passthrough/passthrough.go]: Done. Thanks!

**Comment by mimowo** [comment]
/approve
@mbobrovskyi can we cherry-pick this to website?

**Comment by mbobrovskyi** [comment]
> /approve @mbobrovskyi can we cherry-pick this to website?

I think yes.

**Comment by mbobrovskyi** [comment]
Ah, we need to cherry-pick [this](https://github.com/kubernetes-sigs/kueue/pull/3105) as well...

**Pull request merged**